pyfritzhome/fritzhome.py

```diff
@@ -118,6 +118,8 @@ def get_prefixed_host(self):
 
     def update_devices(self):
         _LOGGER.info("Updating Devices ...")
+        if self._sid is None:
+            raise RuntimeError("Please login before accessing devices")
```

Comment:
Would suggest to define an own exception for that ... something like `NotLogedInException`.
With that, you will be able to handle this specific exception.